website/project/model.py

```diff
@@ -3580,6 +3629,17 @@ def _on_reject(self, user, token):
         )
 
 
+class DraftRegistrationApproval(Sanction):
+
+    mode = Sanction.ANY
+
+    def _on_complete(self, user, token):
+        pass  # draft approval state gets loaded dynamically from this record
```

Comment:
I'm not sure what this comment means. What is the purpose of these methods- can they be removed?